alicorn-utils.lua

```diff
@@ -759,13 +759,7 @@ function M.litprint(s)
 end
 
 function M.debug_break()
```

Comment:
Would `M.debug_break = require("lua-init").debug_break` cause a require loop?

Reply:
No, but requiring internally avoids loading `lua-init` along with `alicorn-utils`, which may be undesired.

Reply:
Shouldn't this be done as a global that is populated with the result of require if it's null?